### ISSUE TITLE
Nested <script> for ckeditor-init.js file

### ISSUE THREAD

**Issue by eriktelepovsky** [django-ckeditor/django-ckeditor#768]
Hello. 

I installed django-ckeditor==6.7.1 and Django==4.2.11 but the CKEditor did not showed up. I noticed weird HTML code generated by CKEditorWidget:

```
<link href="/static/ckeditor/ckeditor.css" media="all" rel="stylesheet">
<script type="text/javascript" src="<script src="/static/ckeditor/ckeditor-init.js" data-ckeditor-basepath="/static/ckeditor/ckeditor/" id="ckeditor-init-script"></script>"></script>
<script type="text/javascript" src="/static/ckeditor/ckeditor/ckeditor.js"></script>
```

Notice nested ``<stript`` in the src for ckeditor-init.js

Any idea please?

**Comment by matthiask** [comment]
This sounds a bit as if you're using `django-js-asset<2` instead of the version which works with Django 4.1 or better?

Recent django-ckeditor versions depend on `django-js-asset>=2.0`, so I'm not sure what's going on.

**Comment by matthiask** [comment]
(... the `CKEDITOR_BASEPATH` thing is necessary when you're using `ManifestStaticFilesStorage` in production.)

**Comment by eriktelepovsky** [comment]
Yes, I have already found some issues and docs related to ``django-js-asset``, ``CKEDITOR_BASEPATH``  and ``ManifestStaticFilesStorage`` but nothing helped.

I have most recent django-js-asset installed:

```bash
pip freeze|grep asset
django-js-asset==2.2.0
```

I already tried setting the ``CKEDITOR_BASEPATH`` to custom value. The path is changed, but the nested script in src is still present.

I don't use any Manifest static file storage. 

And the issue is present on localhost as well.

**Comment by eriktelepovsky** [comment]
No, I use default Django template engine :/
